JoonasHeinonen/escape-from-zordoom-prison
# Feature/theme

Link to the ticket: [Create a theme for the game](https://trello.com/c/KAjunCqH/12-create-a-theme-for-the-game)
![image](https://user-images.githubusercontent.com/33086486/177208988-55df5483-b77e-4ae8-8ca7-591b19311873.png)

**Changes:**
 - Created the initial theme for the menus.
 - Changed the default fonts from `rainyhearts` to `Lato`. Uploaded bold, regular and light.
 - Added custom colors for the buttons.
 - Functionality: Player is able to quit the game as of now.

**How to test:**
 - See that the style is good, is not broken.
 - See to it that it's consistent.

 **Notes and sources:**
  - [Intro to Themes in Godot: User Interface (tutorial)
](https://www.youtube.com/watch?v=g3p3rtdqCDk&ab_channel=GDQuest)
